docs/canvas/canvas-getting-started.asciidoc

```diff
@@ -2,31 +2,158 @@
-. Click the Canvas icon in the left hand navigation menu.
-. Find the *[eCommerce] Revenue Tracking* workpad and click on its name to open it.
+To get up and running with Canvas, use the following tutorial where you'll create a display for monitoring sales at an eCommerce store.
+
+[float]
+=== Before you begin
+
+For this tutorial, you'll use the {kibana-ref}/add-sample-data.html[Sample eCommerce orders data].
+
+[float]
+=== Create and personalize your workpad
+
+Your first step to working with Canvas is to create a workpad. 
+
+. Open *Canvas*.
+
+. Click *Create workpad*.
+
+. To personalize the workpad and page, use the editor on the right.
+
+.. Enter a *Name* for your workpad. For example, `Welcome to Canvas`.
+
+.. Change the page *Background color* to light gray.
+
+[float]
+=== Customize your workpad with images
+
+To customize your workpad to look the way you want, add your own images.
+
+. Click *Add element*, then click *Image*.
++
+The default Elastic logo image appears on your page.
+
+. To replace the Elastic logo with your own image, select the image, then use the editor on the right.
+
+. To move the image, click and drag it to your preferred location.
++
+You'll notice that the image is tagged as an asset, which allows you to reuse the image from *Manage assets*.
+
+[float]
+=== Customize your workpad with text
+
+To express your data with words, add your own text.
+
+. Click *Add element*, then click *Metric*. 
++
+By default, the *Metric* element is connected to a demo data source, which enables you to experiment with the element before you connect it to your own data source.
+
+. To connect the element to your own data source, make sure that the element is selected, then click *Data*.
+
+.. Click *Change your data source*, then click *Elasticsearch SQL*.
+
+.. In the *Elasticsearch SQL query* field, enter `SELECT sum(taxless_total_price) AS sum_total_price FROM "kibana_sample_data_ecommerce"`.
++
+The query selects the total price field and sets it to the sum_total_price field. These fields are pulled from the kibana_sample_data_ecommerce index that you installed.
+
+.. To verify that the data is correct, click *Preview*. If you like what you see, click *Save*.
++
+At this point, the element displays an error.
+
+. Specify how to process and display the data.
+
+.. Click *Display*
+
+.. Under *Number*, select *Value* from the function drop-down list, then select *sum_total_price* from the column drop-down list.
+
+.. Change the *Label* to 'Total sales'.
++
+You'll notice that the error is gone, but the number could use some formatting.
+
+. To format the number, use the Canvas expression language.
+
+.. In the lower right corner, click *Expression editor*.
++
+You're now looking at the raw data syntax that Canvas uses to display the element.
+
+.. Look for `math "sum_total_price"`, then add `| formatNumber "$0a"`.
+
+.. To update the number, click *Run*.
 
 [role="screenshot"]
-image::images/canvas-ecommerce.png[]
```

Comment:
This image should include an image and a metric.  Look at the video to see how to layout the workpad.